pkg/oc/cli/config/loader.go

```diff
@@ -8,15 +8,15 @@ import (
 )
 
 func NewPathOptions(cmd *cobra.Command) *kclientcmd.PathOptions {
-	return NewPathOptionsWithConfig(kcmdutil.GetFlagString(cmd, kclientcmd.OpenShiftKubeConfigFlagName))
+	return NewPathOptionsWithConfig(kcmdutil.GetFlagString(cmd, kclientcmd.RecommendedConfigPathFlag))
 }
 
 func NewPathOptionsWithConfig(configPath string) *kclientcmd.PathOptions {
 	return &kclientcmd.PathOptions{
 		GlobalFile: kclientcmd.RecommendedHomeFile,
 
 		EnvVar:           kclientcmd.RecommendedConfigPathEnvVar,
-		ExplicitFileFlag: kclientcmd.OpenShiftKubeConfigFlagName,
+		ExplicitFileFlag: kclientcmd.RecommendedConfigPathFlag,
```

Comment:
This looks broken.  Please explicitly check both flags with `oc login` and `oc project`

Reply:
I've added test to `login.sh` checking both login and project.